.github/workflows/build.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        go: ['1.20', '1.21']
+        go: ['1.22', '1.23']
```

Comment:
There are new actions for at least `setup-go` (v5.0.2) and `checkout` (v4.1.7) as well.

Reply:
`cache` (v4.0.2) as well.